test/helpers/benchmark.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helpers
+
+import (
+	"fmt"
+
+	"github.com/cilium/cilium/test/ginkgo-ext"
+)
+
+// ClientServerCheck verifies client can still ping server with
+// both IPv4 and IPv6
+func (s *SSHMeta) ClientServerCheck(client string, server string) {
```

Comment:
Why do yo create this helpers here? I do not really liked because: 

- In case of a fail, it'll report in the helper. 
- Is not a helper due is making a assert on that. 


For me this function should be a function into runtime/benchmark.go

Reply:
I'm just going to remove it for now if the netperf fails we can get that from the error there.